projects/media-viewer/src/lib/toolbar/sub-toolbar/sub-toolbar.component.ts

```diff
@@ -49,7 +49,7 @@ export class SubToolbarComponent implements OnInit, OnDestroy {
     this.closeMenu();
   }
 
-  enterIcpMode() {
+  togglePresentBar() {
```

Comment:
if we rename this we should rename it everywhere to be consistent

Reply:
I have?